docs/content/direct/pre-reqs.md

```diff
@@ -1,15 +1,20 @@
 # KubeStellar prerequisites
 
 The following prerequisites are required.
-You can use the [check-pre-req](#check-pre-requisites-for-kubestellar) script, to validate if all needed pre-requisites are installed. 
+You can use the [check-pre-req](#automated-check-of-pre-requisites-for-kubestellar) script, to validate if all needed pre-requisites are installed.
+
+
+## Infrastructure (clusters)
+
+Because of its multicluster architecture, KubeStellar requires that you have the necessary privileges and infrastructure access to create and/or configure the necessary clusters. Our current examples use kind or openshift to do this.
 
 ## For Using KubeStellar
 
-- kubeflex version 0.6.1 or higher
+- **kubeflex** version 0.6.1 or higher
```

Comment:
Actually we are making it so that the KubeFlex CLI, `kflex`, in _not_ required.
@francostellari 
Pointing this out can be done in a separate PR.